packages/react-lowcode/src/codegen/index.ts

```diff
@@ -1,203 +1,4 @@
-import { AppGenerator } from './generation/generators/app-generator'
```

Comment:
Please, do not delete `index.ts`. It is important to have exports here (for easy importing in other projects). No need to have logic here.